script/test-cms-export.js

```diff
@@ -12,19 +14,54 @@ const {
   readEntity,
 } = require('../src/site/stages/build/process-cms-exports/helpers');
 
-const fileNames = readAllNodeNames();
-const entities = map(fileNames, entityDetails =>
-  readEntity(contentDir, ...entityDetails),
+const optionDefinitions = [
+  { name: 'count', alias: 'c', type: Number },
+  { name: 'node', alias: 'n', type: String },
+  { name: 'print', alias: 'p', type: Number },
```

Comment:
I think it would make more sense to [set the `defaultValue`](https://github.com/75lb/command-line-args/blob/master/doc/option-definition.md#optiondefaultvalue--) here rather than use an `|| 0`